io.freetubeapp.FreeTube.metainfo.xml

```diff
@@ -3,15 +3,411 @@
+  <summary xml:lang="en_GB">An open-source YouTube app for privacy</summary>
+  <summary xml:lang="en">An open-source YouTube app for privacy</summary>
+  <summary xml:lang="eo">Malfermfonteca YouTube-aplikaĵo por privateco</summary>
+  <summary xml:lang="es">Una aplicación de YouTube de código abierto para la privacidad</summary>
+  <summary xml:lang="et">Avatud lähtekoodiga YouTube'i rakendus privaatsuse tagamiseks</summary>
+  <summary xml:lang="fa">یک برنامه یوتیوب متن‌باز برای حفظ حریم خصوصی</summary>
+  <summary xml:lang="fi">Avoimen lähdekoodin YouTube-sovellus yksityisyyden suojaamiseksi</summary>
+  <summary xml:lang="fil">Isang open-source na YouTube app para sa privacy</summary>
+  <summary xml:lang="fr">Une application YouTube open source pour la confidentialité</summary>
+  <summary xml:lang="ga">Aip YouTube foinse oscailte le haghaidh príobháideachta</summary>
+  <summary xml:lang="he">אפליקציית YouTube בקוד פתוח לשמירה על פרטיות</summary>
+  <summary xml:lang="hi">गोपनीयता के लिए एक ओपन-सोर्स यूट्यूब ऐप</summary>
+  <summary xml:lang="hr">YouTube aplikacija otvorenog koda za privatnost</summary>
+  <summary xml:lang="hu">Nyílt forráskódú YouTube-alkalmazás az adatvédelem érdekében</summary>
+  <summary xml:lang="hy">Բաց կոդով YouTube հավելված՝ գաղտնիության համար</summary>
+  <summary xml:lang="id">Aplikasi YouTube sumber terbuka untuk privasi</summary>
+  <summary xml:lang="is">Opinn YouTube app fyrir friðhelgi einkalífsins</summary>
+  <summary xml:lang="it">Un'app YouTube open source per la privacy</summary>
+  <summary xml:lang="ja">プライバシー保護のためのオープンソースのYouTubeアプリ</summary>
+  <summary xml:lang="kn">ಗೌಪ್ಯತೆಗಾಗಿ ಓಪನ್ ಸೋರ್ಸ್ YouTube ಅಪ್ಲಿಕೇಶನ್</summary>
+  <summary xml:lang="ko">개인 정보 보호를 위한 오픈 소스 YouTube 앱</summary>
+  <summary xml:lang="lt">Atvirojo kodo „YouTube“ programėlė privatumui užtikrinti</summary>
+  <summary xml:lang="lv">Atvērtā koda YouTube lietotne privātuma nodrošināšanai</summary>
+  <summary xml:lang="nl">Een open-source YouTube-app voor privacy</summary>
+  <summary xml:lang="no_NY">En YouTube-app med åpen kildekode for personvern</summary>
+  <summary xml:lang="oc">Una aplicacion YouTube de còde dobèrt per la vida privada</summary>
+  <summary xml:lang="pa">ਗੋਪਨੀਯਤਾ ਲਈ ਇੱਕ ਓਪਨ-ਸੋਰਸ YouTube ਐਪ</summary>
+  <summary xml:lang="pl">Aplikacja YouTube z otwartym kodem źródłowym zapewniająca prywatność</summary>
+  <summary xml:lang="pt_BR">Um aplicativo de código aberto do YouTube para privacidade</summary>
+  <summary xml:lang="pt">Uma aplicação de código aberto do YouTube para privacidade</summary>
+  <summary xml:lang="ro">O aplicație YouTube open-source pentru confidențialitate</summary>
+  <summary xml:lang="ru">Приложение YouTube с открытым исходным кодом для обеспечения конфиденциальности</summary>
+  <summary xml:lang="si">පෞද්ගලිකත්වය සඳහා විවෘත මූලාශ්‍ර YouTube යෙදුමක්</summary>
+  <summary xml:lang="sk">Aplikácia YouTube s otvoreným zdrojovým kódom pre ochranu súkromia</summary>
```

Comment:
I dont see this anywhere on https://flathub.org/languages. Remove all translations related to this lang